AUMInjector/deobfuscate/2020_12_9s.cpp

```diff
@@ -0,0 +1,9 @@
+#include <deobfuscate.h>
```

Comment:
Why did you move the enum definitions to a seperate cpp file?

Reply:
This was a fix to an issue I was experiencing where I was getting redefinition errors on including the deobfuscation header in both main.cpp **and** GameData.cpp. The pragma once for whatever reason didn't seem to fix it, but I can give it a double-check.

Reply:
![image](https://user-images.githubusercontent.com/26748509/103730837-9ba77b80-4f98-11eb-83f2-695cb8525fbd.png)

This was with a `#pragma once` at the top of the header.
After getting some advice from @ShumWengSang, it looks like the solutions to this problem are moving it to the cpp, using the inline keyword (C++17 and up) or possibly including the deobfuscation header in the PCH for compilation only once?

How would you like to proceed?

Reply:
I see. I'll merge it and test the rest of the code. `__declspec( selectany )` seems to be a promising MSVC alternative to `inline` for older C++ versions.